.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+on:
+  push:
+    branches:
+      - main
+name: release-please
+jobs:
+  release-please-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - id: release-pr
+        uses: GoogleCloudPlatform/release-please-action@v2
+        with:
+          token: ${{ secrets.FORKING_TOKEN }}
+          release-type: node
+          fork: true
+          package-name: googleapis
+          command: release-pr
+      - id: label
+        uses: actions/github-script@v3    
+        with:
+            github-token: ${{secrets.GITHUB_TOKEN}}
+            script: |
+              const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/');
+              const latestRelease = await github.issues.addLabels({
```

Comment:
do we need a `latestRelease` variable if it's not used anywhere? maybe just `await github.issues.addLabels(...)` with no assignment?